src/WinRT.Runtime/IInspectable.net5.cs

```diff
@@ -4,32 +4,30 @@
 using System;
 using System.Collections.Concurrent;
 
-namespace WinRT
-{
+namespace WinRT;

```

Comment:
Nit: is this by accident? As in, it's fixing the namespace in the old runtime for CsWinRT 2.x, not 3.0. We might consider not doing this to avoid merge conflicts maybe, but I don't feel strongly about it.